actions/create_event_handler.go

```diff
@@ -98,9 +110,29 @@ func (h *createEventHandler) Handle(ctx context.Context, action *castai.ClusterA
 	}
 
 	if _, err := h.clientset.CoreV1().Events(event.Namespace).Create(ctx, event, metav1.CreateOptions{}); err != nil {
-		return fmt.Errorf("creating event for ref %v: %w", req.ObjectRef, err)
+		return fmt.Errorf("creating core/v1/event for ref %v: %w", req.ObjectRef, err)
 	}
+	return nil
+}
 
+func (h *createEventHandler) handleEventV1(ctx context.Context, req *castai.ActionCreateEvent, namespace string) error {
+	event := &eventsv1.Event{
```

Comment:
should we also check for similar event here?

Reply:
nope, there are no search API where we can search for the same object and we can't list and filter for all events.

Reply:
Cost are ok with it? Deduplication was done for a reason i believe.

Reply:
right now I am trying to incorporate event recorder that should address duplication issues.